### DISCUSSION TITLE
Changing The Objects Positions

### DISCUSSION THREAD

**Discussion by AntoineUserName** [jrouwe/JoltPhysics.js#187] [Q&A]
Hello,
I need to edit sometimes the position of some objects, I can get the position with the GetPosition() function but I cannot edit the position.
```
    let pos;

    // Do nothing:
    pos = body.GetPosition();
    pos.SetY(8);

    // Do nothing:
    pos = body.GetWorldTransform().GetTranslation();
    pos.SetY(8);
    body.GetWorldTransform().SetTranslation(pos);
```

Note: In this example the variable `body` have the type Jolt.EMotionType_Dynamic.

**Comment by jrouwe** [comment]
`body.GetPosition` / `body.GetWorldTransform` returns a copy of the position. Adjusting it doesn't have any effect other than modifying the copy.

You should use `BodyInterface.SetPosition` to update the position of a body.
